### DISCUSSION TITLE
Deploying Airflow using ArgoCD

### DISCUSSION THREAD

**Discussion by jaruji** [argoproj/argo-cd#17224] [Q&A]
Hello,

I'm deploying an airflow helm chart using ArgoCD. The chart itself gets uploaded correctly and everything seems to work. There is an issue regarding the health and sync status of the argoCD app - after being deployed, it seems to drift out of sync and the health changes to Missing - even though the airflow deployment itself keeps working normally. This seems to be some issue with the way argoCD detects the status of the app. If I enable automated selfHeal, every 5 minutes (more or less), the app status drifts and the job create-user from the airflow chart gets launched. After it is finished executing, the airflow app appears as Healthy and Synced again - only for the cycle to repeat again. Anybody else faced a similar issue and can point me in the right direction?

**Comment by redisded** [comment]
Hi : [https://github.com/apache/airflow/discussions/37470](https://github.com/apache/airflow/discussions/37470) ;-)
